### ISSUE TITLE
can't install r-essentials in base on 2019.10

### ISSUE THREAD

**Issue by kundeng** [ContinuumIO/anaconda-issues#11592]
### Actual Behavior

install anaconda 2019.10
update conda
update --all

then install r-essentials into base env:
(base) griffonuser@griffon:~/Downloads$ conda install r-essentials
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

### Expected Behavior
The documentation seems to suggest this should be completed without any issues. 
In other words, 
* no need to use r or r-forge channel
* no need to add any flags 

##### `conda info`

<details>

```
PASTE OUTPUT HERE:

```

</details>

##### `conda list --show-channel-urls`

<details>

```
PASTE OUTPUT HERE:

```

</details>
